Dockerfile

```
@@ -2,7 +2,7 @@ FROM ubuntu:16.04
 
 # Release parameters
 ENV GOOGLEAPIS_HASH b71d0c74de0b84f2f10a2c61cd66fbb48873709f
-ENV GAPIC_GENERATOR_HASH ea27c8ac15551128e52998a35471711b4c6ffd82
+ENV GAPIC_GENERATOR_HASH 0f4bf83e32d46c5ea242bd065bfb1de56a9d2605
 # Define version number below. The ARTMAN_VERSION line is parsed by
 # .circleci/config.yml and setup.py, please keep the format.
 ENV ARTMAN_VERSION 0.15.9
```

Do you want to release 0.15.10?
```suggestion
ENV ARTMAN_VERSION 0.15.10
```

---

ah yes I did